src/net/sourceforge/kolmafia/textui/command/ColdMedicineCabinetCommand.java

```diff
@@ -1,12 +1,10 @@
 package net.sourceforge.kolmafia.textui.command;
 
-import java.util.List;
-import java.util.Map;
-import java.util.SortedMap;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
Would you please expand the imports instead of collapsing them?  I thought expanding them was a code convention enforced by spotless but apparently not.